src/app/docs/radio/radio-docs.component.html

```diff
@@ -7,8 +7,84 @@
     The radio button module ceates a small set of selection inputs for users to select one option.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
```

Comment:
The current demo demonstrates both reactive and template-driven. Is demonstrating both no longer a priority?

Reply:
The consumers don't have access to the code for the behavioral demo, so it doesn't really matter here. I just went with quick and easy.